profiler/hotapp/main.go

```diff
@@ -135,6 +147,9 @@ func main() {
 	// Simulate some memory allocation.
 	alloc()
```

Comment:
Would it make sense to rename alloc to allocOnce so that it's possible to use a "show stacks" filter with "allocOnce" to filter in just function if that's ever needed?

Reply:
Done!